OrderCloud.Integrations.Payment.CardConnect/Mappers/CardConnectGetSavedCardsMapper.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OrderCloud.Catalyst;
+using OrderCloud.Integrations.Payment.CardConnect.Models;
+
+namespace OrderCloud.Integrations.Payment.CardConnect.Mappers
+{
+    public static class CardConnectGetSavedCardsMapper
+    {
+        public static List<PCISafeCardDetails> ToIntegrationsGetSavedCardsResponse(
+            this List<CardConnectProfile> response)
+        {
+            List<PCISafeCardDetails> savedCards = new List<PCISafeCardDetails>();
+            foreach (CardConnectProfile card in response)
+            {
+                savedCards.Add(new PCISafeCardDetails()
+                {
+                    SavedCardID = card.acctid,
+                    CardType = card.accttype,
+                    ExpirationMonth = card.expiry.Substring(0, 2),
+                    ExpirationYear = card.expiry.Substring(2, 2),
+                    CardHolderName = card.name,
+                    Token = card.token,
+                    NumberLast4Digits = card.token.Substring(card.token.Length - 5, 4)
+                });
+            }
+
+            return savedCards;
+        }
+        public static PCISafeCardDetails ToIntegrationsGetSavedCardResponse(
+            this CardConnectProfile card)
+        {
+            return new PCISafeCardDetails()
+            {
+                SavedCardID = card.acctid,
+                CardType = card.accttype,
+                ExpirationMonth = card.expiry.Substring(0, 2),
+                ExpirationYear = card.expiry.Substring(2, 2),
+                CardHolderName = card.name,
+                Token = card.token,
+                NumberLast4Digits = card.token.Substring(card.token.Length - 5, 4)
```

Comment:
Maybe add a comment that the last 4 digits of a CardConnect token equal the last 4 digits of the card? Because that was unexpected for me and before I learned that I was concerned this was a mistake or a PCI violation.